gusto/common_forms.py

```diff
@@ -194,3 +198,67 @@ def diffusion_form(test, q, kappa):
     form = -inner(test, div(kappa*grad(q)))*dx
 
     return diffusion(form)
+
+
+def split_continuity_form(equation):
+    u"""
+    Loops through terms in a given equation, and splits continuity terms into
+    advective and divergence terms.
+
+    This describes splitting ∇.(u*q) into u.∇q and q(∇.u),
+    for transporting velocity u and transported q.
+
+    Args:
+        equation (:class:`PrognosticEquation`): the model's equation.
+
+    Returns:
+        equation (:class:`PrognosticEquation`): the model's equation.
```

Comment:
The format for the docstrings is a bit different for the Returns section. I think this should be:
```suggestion
        :class:`PrognosticEquation`: the model's equation.
```